src/sx-design/src/Menu/Menu.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+
+import fbt from 'fbt';
+import Icon from '@adeira/icons';
+import React, { useCallback, useState, type Node } from 'react';
+import sx from '@adeira/sx';
+
+import Button from '../Button/Button';
+import MenuItem from './MenuItem';
+import MenuItemDivider from './MenuItemDivider';
+import SxDesignPortal from '../SxDesignPortal';
+
+type AllowedChildren =
+  | RestrictedElement<typeof MenuItem>
+  | RestrictedElement<typeof MenuItemDivider>
+  | $ReadOnlyArray<AllowedChildren>;
+
+type Props = {
+  +children: AllowedChildren,
+};
+
+/**
+ * TODO: render in a `Drawer` for small viewports (see `ModalDrawer`)
+ * TODO: use positioning algorithm similar to `Tooltip` (reuse)
+ * TODO: accessibility (https://www.w3.org/TR/wai-aria-practices-1.1/#menu)
+ */
+export default function Menu(props: Props): Node {
+  const [isMenuOpen, setIsMenuOpen] = useState(false);
+
+  const handleButtonClick = useCallback(() => {
```

Comment:
Why are you using `useCallback` here?

Reply:
"Just in case we are going to optimize `Button` one day" kind of thinking.

But you are right it's probably not a good idea to do it for no reason. Will remove (as well as from the future PRs where I did something simialr).